docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -93,6 +94,7 @@ The following example overrides the `label` style of `Button` in addition to the
 ```jsx
 import React from 'react';
 import Button from '@material-ui/core/Button';
+import './PlainCssButtonDeep.css';
```

Comment:
Would this assume too much about how developer loads the CSS? For Global CSS, I would expect it to fail in the most common cases, won't you get CSS Modules kick-in here? https://github.com/css-modules/css-modules mangling the class names?